setup-gems-cache/action.yml

```
@@ -0,0 +1,32 @@
+name: Setup Gems Cache
+description: Setup Gems Caching for workflow
+inputs:
+  gems-path:
+    description: Homebrew's Ruby gems path
+    required: true
+  gems-hash:
+    description: Homebrew's Ruby Gemfile.lock sha256sum
+    required: true
+outputs:
+  gems-outcome:
+    description: "Brew Install Bundler Gems steps outcome"
```

```suggestion
    description: Brew Install Bundler Gems steps outcome
```

We don't use quotes elsewhere